### DISCUSSION TITLE
❤️ Love or friendship? 🤔

### DISCUSSION THREAD

**Discussion by Vianpyro** [TindETS/.github#2] [Polls]
### Hey team! 👋  

**TindETS** started as a fun idea to connect ETS students through meaningful (or not-so-meaningful) connections. But before we move forward, we want to ensure we’re building something innovative, responsible, and safe for our campus community.

Our original goal was to create an app for **"friendship, serious relationships, casual dates, or even just one night of shared stars."** However, considering the unique dynamics of a school environment—where students often interact daily—we feel it’s important to shape the app’s scope with care and intention.

---

## 🚀 The Question at Hand

How should we define the purpose of **TindETS** to best serve our school community while ensuring safety, inclusivity, and mutual respect?  

Here are the options we’re considering:  

1. **Focus exclusively on friendship and networking**  
   - **Pros**: Creates a safe and supportive space for forming meaningful platonic connections and expanding personal or professional networks.  
   - **Cons**: May not appeal to users seeking romantic or deeper connections.  

2. **Offer options for friendship or relationships (serious or evolving from friendship) - without networking**  
   - **Pros**: Allows users to choose their preferred connection type, appealing to a broader audience while maintaining clarity of intent.  
   - **Cons**: Requires clear boundaries and effective safety measures to manage expectations.  

3. **Include options for friendship, relationships, and one-night connections ("just one night of shared stars") - without networking**  
   - **Pros**: Offers transparency for users seeking short-term, intimate connections, while emphasizing mutual respect and clear expectations.  
   - **Cons**: May attract users primarily interested in casual encounters, potentially disregarding intimacy or respect for other connection types. Requires clear boundaries and robust safety measures to ensure a respectful community.  
 
4. **Other suggestions**: Let us know your thoughts! We’re open to new ideas.  

## Disclaimer

We will always highlight the importance of addressing risks in a shared-campus environment, where interactions could extend beyond the app.  

---

## 📊 Join the Discussion!  

We want to hear from you! Should **TindETS** focus on friendships, relationships, one-night connections, or a mix? Your input will directly influence the app’s direction and its role within our school community.  

Let us know by voting or commenting below. Together, we can create something that reflects our values and ensures everyone feels safe, respected, and included. ❤️

**Comment by MBlueEyed** [comment]
Nice project! Good luck! Be safe everyone ^^
